cloud/deploy/deploy.go

```diff
@@ -561,14 +568,14 @@ func getImageName(cloudDomain, deploymentID, organizationID string, coreClient a
 	}
 
 	currentVersion := resp.JSON200.RuntimeVersion
-	namespace := resp.JSON200.ReleaseName
+	namespace := resp.JSON200.Namespace
 	workspaceID := resp.JSON200.WorkspaceId
 	webserverURL := resp.JSON200.WebServerUrl
-	dagDeployEnabled := resp.JSON200.IsDagDeployEnabled
-	cicdEnforcement := resp.JSON200.ApiKeyOnlyDeployments
+	dagDeployEnabled := resp.JSON200.DagDeployEnabled
+	cicdEnforcement := resp.JSON200.IsCicdEnforced
 	var desiredDagTarballVersion string
-	if resp.JSON200.DesiredDagTarballVersion != nil {
-		desiredDagTarballVersion = *resp.JSON200.DesiredDagTarballVersion
+	if resp.JSON200.DagTarballVersion != nil {
```

Comment:
This check need to remain the same, once we expose this field in v1beta1

Reply:
I agree. Do we have a date for this. CC: @fredzhy